src/server.c

```diff
@@ -201,6 +201,34 @@ mstime_t mstime(void) {
     return ustime()/1000;
 }
 
+mstime_t commandTimeSnapshot(void) {
```

Comment:
let's add some doc comment above the function

Reply:
i added it, please polish it if needed...